client/src/App.js

```diff
@@ -11,6 +11,7 @@ import SearchPage from './pages/SearchPage/SearchPage';
 import ErrorHandler from './containers/ErrorHandler/ErrorHandler';
 import Toaster from './containers/Toaster/Toaster';
 import MapPage from './pages/MapPage/MapPage';
+import CreatePage from "./pages/CreatePage/CreatePage";
```

Comment:
We should probably call this `CreateMonumentPage`

Reply:
Coming in the next commit.